### PR TITLE
Preserve environment from before sourcing enable

### LINKED ISSUE #157: disable_komodo does not reset environment properly
After the recent komodo update disable komodo_does not work properly (most critical is the fact that `LD_LIBRARY_PATH` is not reset properly) 
```bash
$ env > env-dump.txt
$ source /prog/res/komodo/bleeding-py36-rhel7/enable
(bleeding-py36-rhel7) $ disable_komodo
$ env | diff env-dump.txt -
```
gives
```diff
0a1
> MANPATH=/prog/res/komodo/bleeding-py36-rhel7/root/share/man:
4a6
> _PRE_KOMODO_LD_LIBRARY_PATH=
16a19
> LD_LIBRARY_PATH=/prog/res/komodo/bleeding-py36-rhel7/root/lib:/prog/res/komodo/bleeding-py36-rhel7/root/lib64
23a27
> PS1=[\u@\h \W]\$ 
28a33
> _PRE_KOMODO_MANPATH=
```

It seems that the current stable (2020.09.00) does not work perfect either, but at least it resets `LD_LIBRARY_PATH`
```bash
$ env > env-dump.txt
$ source /prog/res/komodo/stable/enable
(2020.09.00-py36) $ disable_komodo
$ env | diff env-dump.txt -
```
gives
```diff
0a1
> MANPATH=/prog/res/komodo/2020.09.00-py36/root/share/man:
4a6
> _PRE_KOMODO_LD_LIBRARY_PATH=
23a26
> PS1=[\u@\h \W]\$ 
28a32
> _PRE_KOMODO_MANPATH=
```

### PR DESCRIPTION
**Pull request by ManInFez**
resolves #157

### CODE REVIEW

File: komodo/data/enable.in [hunk 1]
```diff
@@ -60,7 +72,6 @@ export _PRE_KOMODO_LD_LIBRARY_PATH="$LD_LIBRARY_PATH"
 export LD_LIBRARY_PATH=komodo_prefix/lib:komodo_prefix/lib64${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
 
 export _PRE_KOMODO_PS1="${PS1:-}"
-export PS1="(${KOMODO_RELEASE}) ${PS1:-}"
```
Comment (dotfloat) [inline komodo/data/enable.in:63]: Why are we removing `export PS1`? (best console and environment variable)

File: tests/test_enable_scipts.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        pre_env = json.loads(pre.read())
+    with open("sourced.env") as sourced:
+        sourced_env = json.loads(sourced.read())
+    with open("post_disable.env") as post:
+        post_env = json.loads(post.read())
+
+    return pre_env, sourced_env, post_env
+
+
+TEST_SCRIPT_SIMPLE = """
```
Comment (dotfloat) [inline tests/test_enable_scipts.py:35]: `"""\` with the `\` at the end to avoid an empty first line. ;)

File: komodo/data/enable.in (hunk 1 above)
Comment (ManInFez) [reply komodo/data/enable.in:63]: wooops

**Review by sondreso — APPROVED** [review]
LGTM!

File: tests/test_enable_scipts.py (hunk 2 above)
Comment (ManInFez) [reply tests/test_enable_scipts.py:35]: 👍

**Pull request merged**